packages/expo-updates/src/ExpoUpdatesModule.types.ts

```diff
@@ -0,0 +1,68 @@
+import { ProxyNativeModule } from 'expo-modules-core';
+
+import {
+  Manifest,
+  UpdateCheckResultAvailable,
+  UpdateCheckResultNotAvailable,
+  UpdateCheckResultRollBack,
+  UpdateFetchResultRollBackToEmbedded,
+  UpdateFetchResultFailure,
+  UpdateFetchResultSuccess,
+  UpdatesLogEntry,
+  UpdatesNativeStateMachineContext,
+} from './Updates.types';
+
+export interface ExpoUpdatesModule
+  extends Pick<ProxyNativeModule, 'addListener' | 'removeListeners'> {
+  isEmergencyLaunch?: boolean;
+  isEmbeddedLaunch: boolean;
+  isEnabled: boolean;
+  isUsingEmbeddedAssets?: boolean;
+  /**
+   * Can be empty string
+   */
+  runtimeVersion: string;
+  checkAutomatically: string;
+  /**
+   * Can be empty string
+   */
+  channel: string;
+  shouldDeferToNativeForAPIMethodAvailabilityInDevelopment: boolean;
+  updateId?: string;
+  commitTime?: string;
+  /**
+   * Android
+   */
+  manifestString?: string;
+  /**
+   * iOS
```

Comment:
```suggestion
   * @platform ios
```

https://www.notion.so/expo/Expo-Docs-writing-style-guide-c05e8a2e4a184decbb337d745cdaf7af#146004254a0e492a87a4192a8bae98c9